Traces Panel: Add new Traces Panel visualization (#47534)
grafana/grafana: docs/sources/visualizations/traces.md

> +++

Reviewer: It might be worth taking some of this content into a `README.md` for the plugin, so that the plugin details view has something descriptive for the user. WDYT?

title = "Traces"
keywords = ["grafana", "dashboard", "documentation", "panels", "traces"]
weight = 850
+++

# Traces panel

> **Note:** This panel is currently in beta. Expect changes in future releases.

> _Traces_ are a visualization that allows you to track a single request and log that single request as it crosses through all of the services in your infrastructure.

Reviewer 2: ```suggestion
_Traces_ are a visualization that enables you to track and log a request as it traverses the services in your infrastructure.
```

{{< figure src="/static/img/docs/explore/explore-trace-view-full-8-0.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}

> For more info on traces and how you can set them up, please see the following links

Reviewer 2: ```suggestion
For more information about traces and how to use them, refer to the following documentation:
```

> For more info on traces and how you can set them up, please see the following links

Reviewer 2: I would move the links to the docs above the image.
